com.slack.Slack.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "com.slack.Slack",
+    "base": "io.atom.electron.BaseApp",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "slack",
+    "separate-locales": false,
+    "finish-args": [
+        "--share=ipc", "--socket=x11",
+        "--socket=pulseaudio",
+        "--share=network",
+        "--filesystem=host",
+        "--filesystem=/tmp",
```

Comment:
In my Discord flatpak I set `$TMPDIR` to the private cache dir. In this case you pass through host so I guess security doesn't matter but just a thought.